lte/gateway/c/core/oai/common/sentry_log.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2021 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#ifndef FILE_SENTRY_LOG_SEEN
+#define FILE_SENTRY_LOG_SEEN
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "includes/SentryWrapper.h"
```

Comment:
Is this needed here?

Reply:
looks like it is not, thanks for point this out!